integrationtests/tests/fixtures/mod.rs

```diff
@@ -1025,6 +1044,23 @@ impl FederationTest {
         }
     }
 
+    pub async fn assert_peers_synchronized(&self) {
+        let full_server_status = self.full_server_status().await;
+        println!("assert_peers_synchronized: full_server_status: {full_server_status:?}");
+        assert!(full_server_status
+            .into_iter()
+            .all(|s| s.unwrap().peers_not_synchronized == 0));
+    }
+
+    pub async fn assert_peers_not_synchronized(&self) {
```

Comment:
Isn't this just `!assert_all_peers_synchronized`? Probably not worth having both.